src/containers/User/UserArea.tsx

```diff
@@ -54,16 +56,42 @@ const MainMenuBar: React.FC = () => {
   )
 }
 
-const OrgSelector: React.FC<{ user: User }> = ({ user }) => {
-  // TO-DO: Make into Dropdown so Org can be selected
+const OrgSelector: React.FC<{ user: User; orgs: OrganisationSimple[]; onLogin: Function }> = ({
+  user,
+  orgs,
+  onLogin,
+}) => {
+  const LOGIN_AS_NO_ORG = 0 // Ensures server returns no organisation
+
+  const JWT = localStorage.getItem('persistJWT') as string
+
+  const handleChange = async (_: SyntheticEvent, { value: orgId }: any) => {
+    await attemptLoginOrg({ orgId, JWT, onLoginOrgSuccess })
+  }
+  const onLoginOrgSuccess = async ({ user, orgList, templatePermissions, JWT }: LoginPayload) => {
+    await onLogin(JWT, user, templatePermissions, orgList)
```

Comment:
maybe too much abstraction ?

Reply:
I think it was just to keep parallel structure with what we already did with `onLoginSuccess`.